Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPDistanceTraits.h

```diff
@@ -0,0 +1,106 @@
+/// \ingroup PkgPolygonMeshProcessingConcepts
+/// \cgalConcept
+///
+/// The concept `PMPDistanceTraits` is a refinement of the
+/// concepts `AABBGeomTraits` and `SpatialSortingTraits_3`. In addition to the types required by
+/// those concepts,  it also requires types and functors necessary to
+/// define the functions `approximate_max_distance_to_point_set()`, `sample_triangle_mesh()`,
+/// `approximate_Hausdorff_distance()` and `max_distance_to_triangle_mesh()`
+///
+/// \cgalRefines `AABBGeomTraits` `SpatialSortingTraits_3`
+/// \cgalHasModel Any 3D Kernel is a model of this concept.
+
+class PMPDistanceTraits{
+public:
+    /// Rational traits type
+    /*!
+     * must provide a constructor from `double`
+     */
+    typedef unspecified_type RT;
+    /// A number type
+    /*!
+     * is model of `Field` and `RealEmbeddable`
+     */
+    typedef unspecified_type FT;
+    /// 3D point type
+    /*! There must be a specialisation of CGAL::Kernel_traits for Point_3 and
+     * a function `Point_3 circumcenter(const Point_3&, const Point_3&, const Point_3&)`
+     */
+    typedef unspecified_type Point_3;
+    /// \name Creation
```

Comment:
_3D point type. A constructor from three `FT` must be available. There is a specialization of `CGAL::Kernel_traits` such that `CGAL::Kernel_traits<Point_3>::Kernel` is the model implementing this concept. A_

I don't think other constructors are needed.
Usage of `circumcenter()` must be replaced by `Construct_circumcenter_3`. Point_3 must be constructible from three `FT. Usage of `operator<()` should be replaced by `Less_XYZ_3`.